scripts/appimage.Dockerfile

```diff
@@ -0,0 +1,197 @@
+# -----------------------------------------------------------------------------
+# Base image
+# -----------------------------------------------------------------------------
+FROM ubuntu:24.04 AS base
+ENV DEBIAN_FRONTEND=noninteractive
+ENV CMAKE_GENERATOR=Ninja
+
+# Install only the absolute bare minimum build tools used by almost everyone
+# (wget/curl/ca-certificates for downloading, build-essential for compiling)
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        build-essential \
+        ca-certificates \
+        ccache \
+        cmake \
+        curl \
+        git \
+        gnupg \
+        ninja-build \
+        wget \
+        && rm -rf /var/lib/apt/lists/*
+
+# -----------------------------------------------------------------------------
+# Lib3MF
+# -----------------------------------------------------------------------------
+FROM base AS lib3mf-builder
+
+# Install Lib3MF specific dependencies
+# uuid-dev, libssl-dev etc. are required for Lib3MF
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        uuid-dev \
+        && rm -rf /var/lib/apt/lists/*
+
+# 1.8.1
+ARG LIB3MF_COMMIT=ff914b9fae76cd6a365952728f44f8d79a9f8f75
```

Comment:
Can we bump this to V2 while at it, or are we waiting to resolve the IDL compiler source code issue? @t-paul

Reply:
I think the IDL stuff is only relevant for the upstream packaging, e.g. the Flatpak uses v2.4.1 with a [patch](https://github.com/3MFConsortium/lib3mf/pull/413) that was merged to [v2.5.0-alpha](https://github.com/3MFConsortium/lib3mf/releases/tag/v2.5.0-alpha) but is not released yet. Long story short it's worth trying if it "just builds".